### PR TITLE
Content/pages

### PR DESCRIPTION
**Pull request by Akswii**
Added all the pages i've made so far, still WIP

### CODE REVIEW

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
Just a note for future reference: It is an established convention to prefix feature branches with `feature/` (or `feature-`, so the name of this branch should preferably have been `feature/initial-pages` or something similar. 😃

File: error-page/404/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <span class="topbar-btn-text">Do you need help?</span>
+    </button>
+  </div>
+  <div class="container">
+    <section class="panel panel-default">
+        <div class="panel-body">
+            <h1 class="text-center">404 - This page doesn't exist</h1>
+            <p class="h3 text-center">Oh no, this page doesn't exist. Maybe you misspelled something?</p>
+            <p>&nbsp;</p>
+            <p class="text-center">If you reached this page clicking from anywhere, the owners of that page might appreciate it if you let them know about it. support@payex.com</p>
```
Comment (erikhallaraaker) [inline error-page/404/index.html:24]: Should probably be a mailto link
````
<a href="mailto:support@payex.com">support@payex.com</a>

File: error-page/500/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <span class="topbar-btn-text">Do you need help?</span>
+    </button>
+  </div>
+  <div class="container">
+    <section class="panel panel-default">
+        <div class="panel-body">
+            <h1 class="text-center">500 - internal server error</h1>
+            <p class="h3 text-center">Something went wrong on our end! Try reloading.</p>
+            <p>&nbsp;</p>
+            <p class="text-center">Please contact support at support@payex.com if the problem persists.</p>
```
Comment (erikhallaraaker) [inline error-page/500/index.html:24]: Should probably also be a mailto link.
```
<a href="mailto:support@payex.com">support@payex.com</a>

File: error-page/500/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>This is my Webpage</title>
```
Comment (erikhallaraaker) [inline error-page/500/index.html:6]: Maybe change title? 😄

File: login/index.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Logg inn</title>
```
Comment (erikhallaraaker) [inline login/index.html:6]: Norwegian title and english content?

**Review by erikhallaraaker — COMMENTED** [review]
Overall looks really good. I would change header logo link to payex.com instead of design.payex.com.

**Pull request merged**